axlearn/common/module.py

```diff
@@ -242,6 +242,13 @@ def add_summary(
             name: The name of the item to add.
             value: The value to add.
         """
+
```

Comment:
(Not from this PR, but just noticed that `add_summary` typing seems out of date.)

Reply:
Done.